src/infra/core/src/repos/metadata_chain_impl.rs

```diff
@@ -55,371 +55,15 @@ where
         }
     }
 
-    async fn validate_append_event_logical_structure(
```

Comment:
Oh wut, did we forget to delete this in the previous PR? I though we got rid of this monster already :)

Reply:
During the last PR, we tried hard not to notice this elephant in our virtual room :)